index.bs

```diff
@@ -4963,5 +4988,15 @@ for their contributions as our W3C Team Contacts.
     "href": "https://www.internet2.edu/media/medialibrary/2013/09/04/internet2-mace-dir-eduperson-200604.html",
     "date": "May 15, 2007"
   }
+
```

Comment:
neither of the below references are cited in the above prose re "COSEAlgorithmIdentifier -7 (ES256)", so am wondering why they are here?  Yes, [tokbind-protocol](https://tools.ietf.org/html/draft-ietf-tokbind-protocol-16#section-3.3) cites them, but we are instead citing [[RFC3279]] which seems to obviate citing the former two refs.

seems like we can either cite [[RFC3279]] or do it like tokbind and cite the below.  I suppose either works for me.

Reply:
Removing.